packages/react/src/hooks/utils/query/index.ts

```diff
@@ -1,3 +1,5 @@
 export { useBaseQuery } from './useBaseQuery'
 export { useInfiniteQuery } from './useInfiniteQuery'
 export { useQuery } from './useQuery'
+export { useMutation } from './useMutation'
+export { useQueryClient } from './useQueryClient'
```

Comment:
We should have an eslint rule for this, but we try to alphabetize exports when possible.

```suggestion
export { useBaseQuery } from './useBaseQuery'
export { useInfiniteQuery } from './useInfiniteQuery'
export { useMutation } from './useMutation'
export { useQuery } from './useQuery'
export { useQueryClient } from './useQueryClient'
```